scripts/apps/publish/directives/SubscribersDirective.ts

```diff
@@ -146,13 +146,15 @@ export function SubscribersDirective(
             $scope.saveNewDestination = function() {
                 $scope.destinations.push($scope.newDestination);
                 $scope.newDestination = null;
+                $scope.saveEnabled = true;
```

Comment:
Is there a reason this can't be covered by a `watch`? Save buttons should be enabled/disabled when data changes, not when you explicitly set a variable. Setting a variable might also work for a while, but it's fragile in a way that people need to remember to set the variable. Why not take this complexity away by managing it in a single place?

Reply:
For that, we have to deep watch the destinations collections also, that's why I avoided it.

Reply:
What's wrong with deep watch? Was it too slow?

Reply:
It wasn't too slow. But a deep watch is expensive. I will change it.

Reply:
Test it out, if the expensive operations don't slow the code down, but are required for better code - let's use them, especially when the code is rarely executed. In this case, it will only run when editing subscribers which isn't what users do every day.